k8s/manifests/charts/ck-gateway/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "ck-gateway.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
```

Comment:
We should find a way to not repeat these helpers because they are also in the ck-loadbalancer chart.

Reply:
Yes, you are right. I can probably remove this, it came with creating the initial Chart 👍🏼